src/natcap/invest/urban_flood_risk_mitigation.py

```diff
@@ -458,9 +490,10 @@ def _write_summary_vector(
                     runoff_ret_vol_stats[feature_id]['sum']))
 
         if feature_id in damage_per_aoi_stats:
-            pixel_count = flood_volume_stats[feature_id]['count']
+            pixel_count = runoff_ret_vol_stats[feature_id]['count']
             if pixel_count > 0:
                 damage_sum = damage_per_aoi_stats[feature_id]
+                target_feature.SetField('aff_bld', damage_sum)
                 target_feature.SetField(
                     'serv_bld', damage_sum * float(
                         runoff_ret_vol_stats[feature_id]['sum']))

```

Comment:
Forgive me for going outside the bounds of the changes here, but this equation for `serv_bld` doesn't match the UG's `Service.built`, if I'm looking in the correct place. http://releases.naturalcapitalproject.org/invest-userguide/latest/urban_flood_mitigation.html#calculate-potential-service-optional

Reply:
Yep, you're right that this doesn't match up!  As far as I can tell, the `Service.built` variable noted in the UG is being created as the `Q_m3.tif` output raster.  So things look to be correct within that calculation, despite having misleading variable names.  I'll double-check with Perrine about the name mismatch and the intent of the output vector, in case there's something funky going on here.

Either way, I think we can safely merge this PR since it's consistent with what the model is currently doing, and then I'll follow up with what I find out from Perrine about what the model is supposed to be doing.